### PR TITLE
Translate: Environment Requirements

### PR DESCRIPTION
**Pull request by angelitaooo**
Relates to #4

### CODE REVIEW

File: content/docs/reference-javascript-environment-requirements.md [hunk 1]
```diff
@@ -1,14 +1,17 @@
...
+React 16 depende de tipos de collección [Map]
 
-A polyfilled environment for React 16 using core-js to support older browsers might look like:
+(https://developer.mozilla.org/es/docs/Web/JavaScript/Referencia/Objetos_globales/Map) y [Set]
+(https://developer.mozilla.org/es/docs/Web/JavaScript/Referencia/Objetos_globales/Set). Si soportas navegadores y dispositivos más antiguos que aún no los proporcionan de forma nativa (por ejemplo, IE <11) o que tienen implementaciones no compatibles (por ejemplo, IE 11), considere la posibilidad de incluir un polyfill global en su aplicación empaquetada, como [core-js](https://github.com/zloirock/core-js) o [babel-polyfill](https://babeljs.io/docs/usage/polyfill/).
```
Comment (alejandronanez) [inline content/docs/reference-javascript-environment-requirements.md]: Use `considera` instead of `considere`.

**Comment by tesseralis** [comment]
@alejandronanez the changes were easy enough I went ahead and did them myself. 🙂Will you take a look?

**Comment by carburo** [comment]
@alejandronanez Seems fine for me, it is a small page. I just fixed a typo (collección -> colección) and the inconsistent use of 'environment' (once as 'ambiente', once as 'entorno'). There is another small issue with the use of ['soportar'](https://www.fundeu.es/recomendacion/soporte-soportar/) for tranlationg 'support', but I think we can leave that to the style guide. For me, it's ready to be merged.

**Comment by alejandronanez** [comment]
Say no more @carburo!

Merging!

**Pull request merged**